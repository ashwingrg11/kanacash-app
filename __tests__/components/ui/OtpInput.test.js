import 'react-native';
import React from 'react';

import OtpInputs from '~/components/ui/OtpInputs';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('<OtpInputs/>', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<OtpInputs />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
