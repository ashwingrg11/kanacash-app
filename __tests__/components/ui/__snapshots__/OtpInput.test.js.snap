// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OtpInputs/> renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "borderColor": "#353B39",
            "borderRadius": 5,
            "borderWidth": 1,
            "height": 50,
            "justifyContent": "center",
            "width": 60,
          },
          undefined,
          Object {
            "backgroundColor": "#fff",
          },
        ]
      }
    >
      <TextInput
        allowFontScaling={true}
        keyboardType="numeric"
        onBlur={[Function]}
        onChangeText={[Function]}
        onFocus={[Function]}
        onKeyPress={[Function]}
        rejectResponderTermination={true}
        selectionColor="#fff"
        style={
          Array [
            Object {
              "alignItems": "center",
              "fontSize": 18,
              "fontWeight": "600",
              "textAlign": "center",
            },
            Object {
              "color": "#353B39",
            },
          ]
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Array [
          Object {
            "borderColor": "#353B39",
            "borderRadius": 5,
            "borderWidth": 1,
            "height": 50,
            "justifyContent": "center",
            "width": 60,
          },
          undefined,
          Object {
            "backgroundColor": "#fff",
          },
        ]
      }
    >
      <TextInput
        allowFontScaling={true}
        keyboardType="numeric"
        onBlur={[Function]}
        onChangeText={[Function]}
        onFocus={[Function]}
        onKeyPress={[Function]}
        rejectResponderTermination={true}
        selectionColor="#fff"
        style={
          Array [
            Object {
              "alignItems": "center",
              "fontSize": 18,
              "fontWeight": "600",
              "textAlign": "center",
            },
            Object {
              "color": "#353B39",
            },
          ]
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Array [
          Object {
            "borderColor": "#353B39",
            "borderRadius": 5,
            "borderWidth": 1,
            "height": 50,
            "justifyContent": "center",
            "width": 60,
          },
          undefined,
          Object {
            "backgroundColor": "#fff",
          },
        ]
      }
    >
      <TextInput
        allowFontScaling={true}
        keyboardType="numeric"
        onBlur={[Function]}
        onChangeText={[Function]}
        onFocus={[Function]}
        onKeyPress={[Function]}
        rejectResponderTermination={true}
        selectionColor="#fff"
        style={
          Array [
            Object {
              "alignItems": "center",
              "fontSize": 18,
              "fontWeight": "600",
              "textAlign": "center",
            },
            Object {
              "color": "#353B39",
            },
          ]
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Array [
          Object {
            "borderColor": "#353B39",
            "borderRadius": 5,
            "borderWidth": 1,
            "height": 50,
            "justifyContent": "center",
            "width": 60,
          },
          undefined,
          Object {
            "backgroundColor": "#fff",
          },
        ]
      }
    >
      <TextInput
        allowFontScaling={true}
        keyboardType="numeric"
        onBlur={[Function]}
        onChangeText={[Function]}
        onFocus={[Function]}
        onKeyPress={[Function]}
        rejectResponderTermination={true}
        selectionColor="#fff"
        style={
          Array [
            Object {
              "alignItems": "center",
              "fontSize": 18,
              "fontWeight": "600",
              "textAlign": "center",
            },
            Object {
              "color": "#353B39",
            },
          ]
        }
        underlineColorAndroid="transparent"
      />
    </View>
  </View>
</View>
`;
